lib/scrabble.rb

```diff
@@ -1,7 +1,29 @@
 class Scrabble
 
-  def score(word)
-    1
+  attr_reader :word, :scoring
+
+  def initialize(word)
+    @word = word
+    @scoring = []
+  end
+
+
+  def score
+    #split all the words into characters...=array
+    #for each in array, access the value
+    #add the values and return the score.
+    if @word.length == 1
+      @scoring = point_values[@word.upcase]
```

Comment:
I like that you check if it is single letter, to skip the rest of the logic. Nice guard case.